l10n_de_tax_statement/tests/test_l10n_de_tax_statement.py

```diff
@@ -0,0 +1,363 @@
+                "country_id": self.env.ref("base.de").id,
+                "currency_id": self.eur.id,
+            }
+        )
+        self.env.user.company_id = self.company_parent
+        self.coa.try_loading()
+
+        self.env["l10n.de.tax.statement"].search([("state", "!=", "posted")]).unlink()
+
+        self.tag_1 = self.env["account.account.tag"].create(
+            {
+                "name": "+81 base",
+                "applicability": "taxes",
+                "country_id": self.env.ref("base.de").id,
+            }
+        )
+        self.tag_2 = self.env["account.account.tag"].create(
+            {
+                "name": "+81 tax",
+                "applicability": "taxes",
+                "country_id": self.env.ref("base.de").id,
+            }
+        )
+        self.tag_3 = self.env["account.account.tag"].create(
+            {
+                "name": "+86 base",
+                "applicability": "taxes",
+                "country_id": self.env.ref("base.de").id,
+            }
+        )
+        self.tag_4 = self.env["account.account.tag"].create(
+            {
+                "name": "+86 tax",
+                "applicability": "taxes",
+                "country_id": self.env.ref("base.de").id,
+            }
+        )
+        self.tag_5 = self.env["account.account.tag"].create(
+            {
+                "name": "+41",
+                "applicability": "taxes",
+                "country_id": self.env.ref("base.de").id,
+            }
+        )
+        self.tag_6 = self.env["account.account.tag"].create(
+            {
+                "name": "+21",
+                "applicability": "taxes",
+                "country_id": self.env.ref("base.de").id,
+            }
+        )
+
+        self.tax_1 = self.env["account.tax"].create({"name": "Tax 1", "amount": 22.5})
+        self.tax_1.invoice_repartition_line_ids[0].tag_ids = self.tag_1
+        self.tax_1.invoice_repartition_line_ids[1].tag_ids = self.tag_2
+
+        self.tax_2 = self.env["account.tax"].create({"name": "Tax 2", "amount": 22.5})
+        self.tax_2.invoice_repartition_line_ids[0].tag_ids = self.tag_3
+        self.tax_2.invoice_repartition_line_ids[1].tag_ids = self.tag_4
+
+        self.statement_1 = self.env["l10n.de.tax.statement"].create(
+            {"name": "Statement 1", "version": "2018"}
+        )
+
+    def _create_test_invoice(self):
+        journal = self.env["account.journal"].create(
+            {"name": "Journal 1", "code": "Jou1", "type": "sale"}
+        )
+        partner = self.env["res.partner"].create({"name": "Test partner"})
+        account_receivable = self.env["account.account"].create(
+            {
+                "user_type_id": self.env.ref("account.data_account_type_expenses").id,
+                "code": "EXPTEST",
+                "name": "Test expense account",
+            }
+        )
+        invoice_form = Form(
+            self.env["account.move"].with_context(default_type="out_invoice")
```

Comment:
This should fix the error in the test you mentioned:

```suggestion
            self.env["account.move"].with_context(default_move_type="out_invoice")
```